test/integration/generated/mediaTypesV3Lro/src/mediaTypesV3LROClient.ts

```diff
@@ -56,6 +56,29 @@ export class MediaTypesV3LROClient extends coreClient.ServiceClient {
       baseUri: options.endpoint ?? options.baseUri ?? "{$host}"
     };
     super(optionsWithDefaults);
+
+    if (options?.pipeline && options.pipeline.getOrderedPolicies().length > 0) {
+      const pipelinePolicies: coreRestPipeline.PipelinePolicy[] = options.pipeline.getOrderedPolicies();
```

Comment:
```suggestion
      const pipelinePolicies = options.pipeline.getOrderedPolicies();
```

Reply:
Same as last one since this is generated from the last code